src/cf-forms/usage.md

```diff
@@ -380,3 +380,19 @@ creating a typical site search form.
     </div>
 </div>
 ```
+
+### Basic Multi-select
+
+<div class="m-form-field m-form-field__select">
+    <label class="a-label" for="test_select__multiple">Label</label>
+    <select class="a-multiselect" id="test_select__multiple" multiple>
+        <option value="option1">Option 1</option>
+        <option value="option2">Option 2</option>
+        <option value="option3">Option 3</option>
+        <option value="option4">Option 4</option>
+        <option value="option1">Option 5</option>
+        <option value="option2">Option 6</option>
+        <option value="option3">Option 7</option>
+        <option value="option4">Option 8</option>
+    </select>
+</div>
```

Comment:
It needs this code as a code block below it as well right? Like line 382 and above https://github.com/cfpb/capital-framework/pull/490/files#diff-5c37fcdebfa9194bc311e39c264c7154R382 ?